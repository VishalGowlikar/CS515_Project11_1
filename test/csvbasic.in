Name,Age,Salary,Department,Location
Liam,27,65000,Software Development,San Francisco
Olivia,31,61000,Data Science,Seattle
Noah,29,60000,Product Management,San Jose
Emma,28,59000,IT Operations,Austin
Mason,30,62000,Cybersecurity,New York