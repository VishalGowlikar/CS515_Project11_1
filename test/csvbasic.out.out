Sums of specified columns: 145.0